ADOPTERS.md

```diff
@@ -206,6 +206,11 @@ production users that have added themselves (in alphabetical order):
   across a fleet of microservices that evolved out of a monolithic architecture.
   For more information see this talk from [KubeCon US 2019](https://www.youtube.com/watch?v=Z6aN3Smt-9M).
 
+* [VNG Cloud](https://www.vngcloud.vn/en/home) [Identity and Access Management (IAM)](https://iam.vngcloud.vn/)
```

Comment:
Since this list is ordered alphabetically, this entry should be moved up to be before "Wiz" above.

Reply:
I have updated it as you suggested.